### PR TITLE
feat: Added warning note to all public endpoints 

### PR DESCRIPTION
**Pull request by vara-bonthu**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### Motivation

- Added warning note to all the public endpoints related to VPC, EKS and ELB to ensure it meets the security requirements.

### More

- [ ] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: ai-ml/jark-stack/terraform/helm-values/ingress-nginx-values.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ controller:
       service.beta.kubernetes.io/aws-load-balancer-type: external
       service.beta.kubernetes.io/aws-load-balancer-nlb-target-type: ip
       service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
-      service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing
+      service.beta.kubernetes.io/aws-load-balancer-scheme: internal # Private Load Balancer can only be accessed within the VPC
```
Comment (askulkarni2) [inline ai-ml/jark-stack/terraform/helm-values/ingress-nginx-values.yaml:8]: This will break the blog. Why are we doing this? Can't we just add a comment?

Comment (lusoal) [reply ai-ml/jark-stack/terraform/helm-values/ingress-nginx-values.yaml:8]: Can we change the blog to do port-forward instead of using LB?

**Review by lusoal — COMMENTED** [review]
LGTM

**Review by askulkarni2 — APPROVED** [review]
LGTM!

File: website/docs/blueprints/data-analytics/datahub-on-eks.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ chmod +x install.sh
 
 ### Verify Deployment
 
-After the deployment completes, we can access the DataHub UI and test importing metadata from sample datasources.  For demo purpose, this blueprint creates the Ingress object for the datahub FrontEnd UI with public LoadBalancer(internet-facing).  For production workloads, you can modify datahub_values.yaml to use internal LB:
+After the deployment completes, we can access the DataHub UI and test importing metadata from sample datasources.  For demo purpose, this blueprint creates the Ingress object for the datahub FrontEnd UI with public LoadBalancer(internal # Private Load Balancer can only be accessed within the VPC).  For production workloads, you can modify datahub_values.yaml to use internal LB:
```
Comment (Libbum) [inline website/docs/blueprints/data-analytics/datahub-on-eks.md:81]: This was probably a search/replace, so this documentation got changed in an odd manner.

Comment (vara-bonthu) [reply website/docs/blueprints/data-analytics/datahub-on-eks.md:81]: Thank you👍 We will update the docs

**Pull request merged**